vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.targets

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- AutolinkedNativeModules.g.targets contents generated by "react-native autolink-windows" -->
+  <ItemGroup>{{ &autolinkProjectReferencesForTargets }}
```

Comment:
If I read the doc correctly it sounds like & is used to render unescaped HTML, meaning that by default it escapes things as if they were to end up in HTML. That's a weird default for mustache...

Should we standardize on how we use mustache for unescaped text? it looks like you can also use {{{ }}} (triple braces) which might be slightly less eye-sore-y :)

Reply:
Mustache was for templating HTML, which is why it escapes by default. Since we're using it to template code, especially code in a language that uses curly braces, adding another set of curly braces makes it harder to spot. I'd rather have an explicit thing you can see (there are other characters used by mustache for other things like ^ conditionals).

Since the decision was made to use mustache everyone needs to learn mustache.